### PR TITLE
Technical PR

### PR DESCRIPTION
**Pull request by JulianVolodia**
Updated ReadMe. Not my opinion but next parts should be moved to Wiki.

Updated and added Docker scripts.

Small bug from views (#224) - it probably never happen.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -45,3 +41,12 @@ RUN mkdir /root/.ssh/
 RUN touch /root/.ssh/environment
 
 CMD env >> /root/.ssh/environment; export -p | grep _ >> /etc/profile; /usr/sbin/sshd -D;
+
+# settings default django settings module (2nd part)
+ENV POSTGRES_PORT_5432_TCP_PORT="5432"
+ENV POSTGRES_PORT_5432_TCP_ADDR="172.17.0.2"
```
Comment (ppeczek) [inline Dockerfile]: why? docker assigns different address each time

File: docker_bye.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+NAME='politikon'
+DB_NAME='politikon_db'
+PSQL_VER='postgres:9.4.1'
```
Comment (ppeczek) [inline docker_bye.sh]: latest

**Comment by ppeczek** [comment]
nice shell scripts

File: politikon/views.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ def get_context_data(self, *args, **kwargs):
         for event in home_events:
             event.my_bet = event.get_user_bet(user)
 
-        front_event = home_events[0]
+        front_event = home_events[0:1]
```
Comment (ppeczek) [inline politikon/views.py]: +:1+

**Comment by ppeczek** [comment]
Divide it for smaller PRs, please. This PR refers to 3 different issues :disappointed:

File: docker_bye.sh (hunk 2 above)
Comment (JulianVolodia) [inline docker_bye.sh]: ouch... thanks!

File: Dockerfile (hunk 1 above)
Comment (JulianVolodia) [inline Dockerfile]: I will investigate if it needed but me and @WojtekReu tested that it's needed and then works... (but with less iterations... not for each env var).

File: docker_bye.sh (hunk 2 above)
Comment (JulianVolodia) [inline docker_bye.sh]: btw.  they could be better but now we could use them

**Comment by JulianVolodia** [comment]
@ppeczek I know but you think is it necessary now really? you want have more reverting possibility yep?

File: Dockerfile (hunk 1 above)
Comment (JulianVolodia) [inline Dockerfile]: https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/#environment-variables

They seem to be static then

**Comment by JulianVolodia** [comment]
also better description is wanted?

**Pull request merged**